jdisc_core/pom.xml

```diff
@@ -167,6 +167,12 @@
             <version>${project.version}</version>
             <scope>compile</scope>
         </dependency>
+        <dependency>
+            <groupId>com.yahoo.vespa</groupId>
+            <artifactId>annotations</artifactId>
+            <version>${project.version}</version>
+            <scope>provided</scope>
```

Comment:
'provided' deps do not make sense for jdisc_core, as it's on the classpath. For this to be usable, it must be compile, and the packages re-exported via jdisc_core.jar.